modules/python/CMakeLists.txt

```diff
@@ -3,30 +3,30 @@
 # ----------------------------------------------------------------------------
 if(DEFINED OPENCV_INITIAL_PASS)  # OpenCV build
 
-if(ANDROID OR APPLE_FRAMEWORK OR WINRT)
-  ocv_module_disable_(python3)
-  return()
-elseif(BUILD_opencv_world OR (WIN32 AND CMAKE_BUILD_TYPE STREQUAL "Debug"))
-  if(NOT DEFINED BUILD_opencv_python3)
-    set(__disable_python3 ON)
+  if(ANDROID OR APPLE_FRAMEWORK OR WINRT)
+    ocv_module_disable_(python3)
+    return()
+  elseif(BUILD_opencv_world OR (WIN32 AND CMAKE_BUILD_TYPE STREQUAL "Debug"))
+    if(NOT DEFINED BUILD_opencv_python3)
+      set(__disable_python3 ON)
+    endif()
   endif()
-endif()
 
-add_subdirectory(bindings)
+  add_subdirectory(bindings)
 
-add_subdirectory(test)
+  add_subdirectory(test)
 
-if(__disable_python3)
-  ocv_module_disable_(python3)
-  return()
-endif()
+  if(__disable_python3)
+    ocv_module_disable_(python3)
+    return()
+  endif()
 
-add_subdirectory(python3)
+  add_subdirectory(python3)
 
 else()  # standalone build
 
-cmake_minimum_required(VERSION 3.13)
-project(OpenCVPython CXX C)
-include("./standalone.cmake")
```

Comment:
I propose not touch it as there is no functional changes. It leads to many 4.x->5.x merge collisions.